### PR TITLE
add admin server helm chart

### PR DESCRIPTION
**Pull request by bellatoris**

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #109** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/109" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/109?utm_source=stack-comment).

**Review by Zeniuus — APPROVED** [review]
기존 infra/k8s/dev 아래의 파일들은 삭제?

-> 아 #111 에 있구나

File: infra/helm/scc-admin-frontend/values.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Default values for scc-admin-frontend.
```
Comment (Zeniuus) [inline infra/helm/scc-admin-frontend/values.yaml:1]: 이 파일도 삭제해도 될 것 같네욤

Comment (Zeniuus) [reply infra/helm/scc-admin-frontend/values.yaml:1]: 아 #111 에 있구나

**Comment by bellatoris** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/109?utm_source=stack-comment).


Job Reference: XhQAUDpWA2PpvPF9lhCY

**Pull request merged**